Make.inc

```diff
@@ -815,12 +819,26 @@ OPENBLAS_DYNAMIC_ARCH:=0
 OPENBLAS_TARGET_ARCH:=ARMV8
 endif
 
+do_step =
 # Set MARCH-specific flags
 ifneq ($(MARCH),)
 CC += -march=$(MARCH)
 CXX += -march=$(MARCH)
 FC += -march=$(MARCH)
 JULIA_CPU_TARGET ?= $(MARCH)
+do_step = yes
+endif
+
+# Set MCPU-specific flags
+ifneq ($(MCPU),)
+CC += -mcpu=$(MCPU)
+CXX += -mcpu=$(MCPU)
+FC += -mcpu=$(MCPU)
+JULIA_CPU_TARGET ?= $(MCPU)
+do_step = yes
+endif
+
+ifdef do_step
```

Comment:
There might be a better way to do the OR logic here

Reply:
```suggestion
ifneq ($(MARCH)$(MCPU),)
```

Reply:
Great, thanks. Incorporated